scripts/enable-ecs-agent-inferentia-support.sh

```diff
@@ -34,6 +34,9 @@ sudo yum install -y aws-neuron-runtime-base
 
 sudo yum install -y pciutils oci-add-hooks
 
+# disable neuron package upgrades by deleting the yum repo
+sudo rm /etc/yum.repos.d/neuron.repo
```

Comment:
Curious: Why not use `yum-config-manager` to disable the repo instead of deleting it?

Reply:
because we've already written out the repo file manually in the steps above....and actually it seems that `yum-config-manager` is not installed by default on AL2 instances:
```
% sudo yum-config-manager --disable neuron
sudo: yum-config-manager: command not found
```

Reply:
I see!! Thanks for clarifying:)